src/Segment.php

```diff
@@ -27,6 +28,7 @@ public function __construct($m3u8Version = null)
         $this->extinfTag = new Tag\ExtinfTag($m3u8Version);
         $this->byteRangeTag = new Tag\ByteRangeTag();
         $this->discontinuityTag = new Tag\DiscontinuityTag();
+        $this->streamTags = new StreamTags();
```

Comment:
according to 'https://tools.ietf.org/html/rfc8216#section-4.3.4.2', one stream inf tag should have one URI part. so I think `StreamTags` does not belong to a `Segment`, `StreamTag` itself is a `Segment`.

```php
$this->streamTag = new StreamTag();
```

Yes, `Segment` does not sound proper, but we can just define 'Segment' as 'a URI with several tags with it', no matter it's Media or Master playlist.